lib/slack-ruby-bot-server.rb

```diff
@@ -1,15 +1,22 @@
 require 'celluloid/current'
-require 'kaminari/grape'
-require 'mongoid-scroll'
+
 require 'grape-swagger'
 require 'slack-ruby-bot'
+require 'slack-ruby-bot-server/server'
+require 'slack-ruby-bot-server/config'
 
 require 'slack-ruby-bot-server/ext'
 require 'slack-ruby-bot-server/version'
 require 'slack-ruby-bot-server/info'
-require 'slack-ruby-bot-server/models'
+
+if SlackRubyBotServer::Config.mongo?
```

Comment:
Create two files, maybe `config/databases/mongo.rb` and `config/databases/postgresql.rb` and move all this logic there. The require should be something like `require 'config/databases/SlackRubyBotServer::Config.database_type`.